File: mobile/iOS/GuestJini/GuestJini/Global/Model/Responses/KBRatingResponse.swift
//
//  KBRatingResponse.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 22/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class KBRatingResponse: Decodable {
    public var kbRating:KBRating? = nil
    public var kbRatingList:[KBRating]?  = nil
    public var totalPages:Int = 0
    public var totalRecords:Int = 0
    public var currentRecords:Int = 0
    public var success:Bool = false
    public var error:String = ""
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/KnowledgeBase/KBRatingResponse.java
package com.carmel.guestjini.Networking.KnowledgeBase;

import java.util.List;

public class KBRatingResponse {
    private KBRating kbRating;
    private List<KBRating> kbRatingList;
    private int totalPages;
    private int totalRecords;
    private int currentRecords;
    private Boolean success;
    private String error;

    public KBRating getKbRating() {
        return kbRating;
    }

    public void setKbRating(KBRating kbRating) {
        this.kbRating = kbRating;
    }

    public List<KBRating> getKbRatingList() {
        return kbRatingList;
    }

    public void setKbRatingList(List<KBRating> kbRatingList) {
        this.kbRatingList = kbRatingList;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(int totalPages) {
        this.totalPages = totalPages;
    }

    public int getTotalRecords() {
        return totalRecords;
    }

    public void setTotalRecords(int totalRecords) {
        this.totalRecords = totalRecords;
    }

    public int getCurrentRecords() {
        return currentRecords;
    }

    public void setCurrentRecords(int currentRecords) {
        this.currentRecords = currentRecords;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}
